docs/products/flink/howto/rest-api-cheatsheet.rst

```diff
@@ -0,0 +1,73 @@
+Flink REST API examples cheatsheet
+==================================
+
+This page contains an overview of the REST API calls examples useful to create Aiven for Apache Flink table and job definitions.
+
+Create PostgreSQL based Flink table
+-------------------------------------------------
+
+Define a Flink table named ``node_info``
+
+* pointing to a PostgreSQL table named ``public.node_definition`` available in the PostgreSQL database identified by the integration with id ``a4af7409-d167-4f31-af13-ddd4cd06564f``. 
```

Comment:
I think "id" could be capitalized here and preceded by an article -> "...integration with the ID..."

Reply:
Perfect, fixed!